README.md

```diff
@@ -58,5 +58,5 @@ pip install -e .
 ```
 * Run the tests
 ```
-python setup.py test
+CLRENV_PATH=tests/env.yaml python setup.py test
```

Comment:
we can get rid of this now ya?

Reply:
and the file

Reply:
It doesn't seem like we can as I'm _guessing_ clrenv searches for the env file on import.